torchdynamo/symbolic_convert.py

```diff
@@ -1391,7 +1422,7 @@ def inline_call(parent, func, args, kwargs):
         if getattr(func, "__closure__", None) is not None:
             unimplemented("inline with  __closure__")
         if getattr(func, "__self__", None) is not None:
-            unimplemented("inline with  __self__")
+            func = func.__func__
```

Comment:
Why don't we need to add an extra arg for self here.  Inlining won't work without that.